conu/backend/docker/container.py

```diff
@@ -475,3 +490,69 @@ def write_to_stdin(self, message):
             self.popen_instance.stdin.flush()
         except subprocess.CalledProcessError as e:
             raise ConuException(e)
+
+    def get_container_metadata(self):
+        """
+        Convert dictionary returned after docker inspect command into instance of ContainerMetadata class
+        :return: ContainerMetadata, container metadata instance
+        """
+
+        docker_metadata = self.get_metadata(refresh=True)
+
+        print(docker_metadata)
+
+        # format of Environment Variables from docker inspect:
+        # ['DISTTAG=f26container', 'FGC=f26']
+        env_variables = dict()
+        for env_variable in docker_metadata['Config']['Env']:
+            env_variables.update({env_variable.split('=')[0]: env_variable.split('=')[1]})
```

Comment:
I advise to do `.split("=", 1)` since `=` can be put also in the value; the other potential error could occur if the value is unset

Please add these two cases to tests.

Real example:
```
            "Env": [
                "ASD=",
                "A=B=C=D",
                "XYZ",
                "DISTTAG=f28container",
                "FGC=f28"
            ],
```

```
$ env
A=B=C=D
HOSTNAME=36256ed1fe0d
PWD=/
HOME=/root
DISTTAG=f28container
ASD=
FGC=f28
TERM=xterm
SHLVL=1
PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
_=/usr/bin/env
```